.github/workflows/ci_dockers.yml

```diff
@@ -2,11 +2,23 @@ name: CI build Docker
 # https://www.docker.com/blog/first-docker-github-action-is-here
 # https://github.com/docker/build-push-action
 # see: https://help.github.com/en/actions/reference/events-that-trigger-workflows
-on:  # Trigger the workflow on push or pull request, but only for the master branch
+on: # Trigger the workflow on push or pull request, but only for the master branch
   push:
-    branches: [master, "release/*"]  # include release branches like release/1.0.x
+    branches: [master, "release/*"] # include release branches like release/1.0.x
+    paths:
+      - "dockers/**"
+      - "!dockers/README.md"
+      - "requirements/*"
+      - "environment.yml"
+      - "requirements.txt"
```

Comment:
this can be a problem if we deploy auto-merge and this will yield a different number if triggered tests...

Reply:
I think auto-merge only needs `Required` CI to pass and the number of review by core (3 in this repo.)

Reply:
depends on which tool is used, but for example, mergify also has/sets the number of checked passed

Reply:
I see, but with github PR auto-merge, I think it would be ok